rmgpy/molecule/resonance.py

```diff
@@ -182,11 +182,12 @@ def generate_resonance_structures(mol, clar_structures=True, keep_isomorphic=Fal
                       " definition.".format(mol.to_adjacency_list()))
         raise
     if mol.get_net_charge() != 0:
-        raise ValueError("Got the following structure:\nSMILES: {0}\nAdjacencyList:\n{1}\nNet charge: {2}\n\n"
-                         "Currently RMG cannot process charged species correctly."
-                         "\nIf this structure was entered in SMILES, try using the adjacencyList format for an"
-                         " unambiguous definition.".format(mol.to_smiles(), mol.to_adjacency_list(), mol.get_net_charge()))
-
+        # logging.info("Got the following structure:\nSMILES: {0}\nAdjacencyList:\n{1}\nNet charge: {2}\n\n"
```

Comment:
is this staying commented out? or do we want it logged?

Reply:
Logging it would probably be best.  I cherrypicked this commit from my electrocat branch where we do get charged species, but don't generate resonance structs for them (just return original molecule:  [mol])

Reply:
I think leaving it commented out is fine.  Eventually we will be able to processed charged species and don't want to log this every time